packages/toolkit/src/normalizers.js

```diff
@@ -10,15 +10,15 @@
 // Note this is only used when you need to deal with CKB structures in molecule
 // format. If you are using RPCs or GraphQL to interact with CKB, chances are you
 // will not need this package.
-import JSBI from "jsbi";
+import { BI } from "@ckb-lumos/bi";
 import { Reader } from "./reader";
 /**
  * @deprecated please follow the {@link https://lumos-website.vercel.app/migrations/migrate-to-v0.19 migration-guide}
  */
 function normalizeHexNumber(length) {
   return function (debugPath, value) {
     if (!(value instanceof ArrayBuffer)) {
-      let intValue = JSBI.BigInt(value).toString(16);
+      let intValue = BI.from(value).toHexString();
```

Comment:
fixed and added test case by [fe92d3e](https://github.com/ckb-js/lumos/pull/560/commits/fe92d3eb587483ab3ce319113f56511fbabc7b05), the motivation is that since we are implementing our own Bignumber lib in lumos, why not use is other than use third-party lib. Another reason is that peer dependency needs more configuration when used by other projects if they are using yarn as package manager https://github.com/nervosnetwork/neuron/pull/2843#issuecomment-1730907381 , if we remove peer dependency, the problem will be gone for other projects too